src/css/style.css

```diff
@@ -688,3 +688,138 @@ textarea {
     background-position: top right;
   }
 }
+
+/* Modal styles */
+.modal-container {
+  display: none;
+}
+
+.modal {
+  position: fixed;
+  background-color: rgba(128, 128, 128, 0.9);
+  inset: 0;
+  z-index: 10000;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
```

Comment:
- [ ] Great job on working on the requested changes but I think while fixing the desktop version you made something wrong in the pop up for the mobile version I noticed that you need to decrease the height a little bit so I can see the whole card because I can't see the X button and the buttons for see project and live version. In my opinion you can decrease the description a little bit you just need small abstract about each project 😊💯🚀🔥


- Your design
![image](https://user-images.githubusercontent.com/58553711/188764392-2dcf94b2-d7be-4210-bde9-8c6efc9f012d.png)

- Figma design
![image](https://user-images.githubusercontent.com/58553711/188764566-b6086b92-3ce8-424c-9f27-aa88eec10658.png)